pkg/api/aim/query/query_test.go

```diff
@@ -42,47 +42,95 @@ func (s *QueryTestSuite) TestPostgresDialector_Ok() {
 		expectedVars []interface{}
 	}{
 		{
-			name:         "TestRunNameWithoutFunction",
-			query:        `(run.name == 'run')`,
-			expectedSQL:  `SELECT * FROM "runs" WHERE ("runs"."name" = $1 AND "runs"."lifecycle_stage" <> $2) ORDER BY "runs"."run_uuid" LIMIT 1`,
+			name:  "TestRunNameWithoutFunction",
+			query: `(run.name == 'run')`,
+			expectedSQL: `SELECT "run_uuid" FROM "runs" ` +
```

Comment:
Jonathan it is hard to maintain all this `+` Why do we need them? We already use `backticks`.

Reply:
the idea was to make this more readable, and to avoid lines that are too long

Reply:
correct. but `backticks` allows you to use new lines.

Reply:
yes but we don't want new lines to be part of the string!